tests/lib/nested.sh

```diff
@@ -903,7 +903,8 @@ nested_start_core_vm() {
     else
         # Start the nested core vm
         nested_start_core_vm_unit "$CURRENT_IMAGE"
-    fi    
+    fi
+    nested_wait_for_snap_command
```

Comment:
Could you please propose this separately.

Reply:
sure